tests/functional/z_cluster/test_multiple_mds.py

```diff
@@ -0,0 +1,192 @@
+"""
+Test cases for multiple mds support
+"""
+
+import logging
+import random
+import time
+
+import pytest
+
+from ocs_ci.framework.pytest_customization.marks import brown_squad, tier4c
+from ocs_ci.helpers.helpers import verify_storagecluster_nodetopology
+from ocs_ci.framework import config
+from ocs_ci.ocs.cluster import (
+    adjust_active_mds_count_storagecluster,
+    get_active_mds_count_cephfilesystem,
+    get_active_mds_pods,
+    get_active_and_standby_pod_count,
+)
+from ocs_ci.ocs.resources import pod
+from ocs_ci.helpers.sanity_helpers import Sanity
+from ocs_ci.ocs import node
+from tests.functional.z_cluster.nodes.test_node_replacement_proactive import (
+    delete_and_create_osd_node,
+)
+
+
+log = logging.getLogger(__name__)
+
+
+@brown_squad
+@tier4c
+class TestMultipleMds:
+    """
+    Tests for support multiple mds
+
+    """
+
+    @pytest.fixture(autouse=True)
+    def teardown(self, request):
+        """
+        Make sure mds pod count is set to original.
+
+        """
+
+        def finalizer():
+            adjust_active_mds_count_storagecluster(
+                1
+            ), "Failed to set active mds count to 1"
+
+        request.addfinalizer(finalizer)
+
+    @pytest.fixture(autouse=True)
+    def init_sanity(self):
+        """
+        Initialize Sanity instance
+
+        """
+        self.sanity_helpers = Sanity()
+
+    def test_multiple_mds(self, cluster):
```

Comment:
you are also doing the node replacement, please update the test name reflecting the same